Add notice about rmw_connextdds (#492)
ros2/rmw_connext: README.md

# rmw_connext
Implementation of the ROS Middleware (rmw) Interface using [RTI's Connext DDS](https://www.rti.com). 

> **DEPRECATION NOTICE:** [rmw_connextdds](https://github.com/ros2/rmw_connextdds)

Author: No sure if "Deprecation Notice" is the correct term. Please feel free to suggest better wording, thanks!

Reviewer: Migration Notice?

Reviewer 2: Deprecation Notice is fine with me.

Author: I'll leave it as "Deprecation Notice"

is a new RMW implementation for RTI Connext DDS, which supersedes the one
contained in this repository (`rmw_connext_cpp`). This new implementation was
developed by RTI in collaboration with the ROS 2 community, and it resolves

> several performance issue that were present in the previous implementation.

Reviewer 2: ```suggestion
several performance issues that are present in this implementation.
```

> `rmw_connextdds` is already available in Rolling, and it will be included in
> future ROS 2 releases starting with Galactic.

Reviewer 2: Thinking towards the future, I'd just say:

```suggestion
`rmw_connextdds` is included in ROS 2 releases starting with Galactic.
This rmw implementation will be supported until the end-of-life of the ROS distributions it is available in (ROS 2 Dashing and Foxy).
```
